frank-doc-doclet/src/main/java/org/frankframework/frankdoc/Utils.java

```diff
@@ -222,36 +231,72 @@ public static String jsonPretty(String json) {
 	}
 
 	public static String flattenJavaDocLinksToLastWords(String text) throws FrankDocException {
+		return replacePattern(text, JAVADOC_LINK_START, s -> getLinkReplacement(s));
+	}
+
+	private static String replacePattern(String text, String patternStart, Function<String, String> substitution) throws FrankDocException {
```

Comment:
Ik vind het onduidelijk dat dit replacePattern heet, maar niet doet. 
Je geeft er alleen een start delimiter aan mee, en gaat er van uit dat de end delimiter '}' is.
Het is niet het pattern dat gereplaced wordt, maar de contents na het pattern, tot aan '}'.

Reply:
Volgens mij vervang je wél het hele pattern. Ook de `{@` en de `}` characters worden door deze method vervangen. Het functional object dat je meegeeft vervangt de text die tussen `{@link ...}` of `{@value ...}` staat, maar `replacePattern()` vervangt het hele patroon.

Reply:
@gvanbrakel, we hebben het erover gehad. Ik heb twee punten verbeterd:
* De delimiters die je gebruikt zijn static final velden. Ik heb hun naam aangepast zodat het woord DELIMITER erin zit.
* Ik heb van de end delimiter ook een argument gemaakt.